web/templates/widgets/todo_widget_tasks.hbs

```diff
@@ -8,9 +8,9 @@
             </div>
             <div>
                 {{#if completed}}
-                <strike><strong>{{ task }}</strong>{{#if desc }}: {{ desc }}{{/if}}</strike>
+                <strike><strong class="task_complete">{{{ task }}}</strong>{{#if desc }}<span class="task_desc_complete"">: </span><span class="description_complete"">{{{ desc }}}</span>{{/if}}</strike>
```

Comment:
There's an extra `"` after the `task_desc_complete` class... Also, can you explain why these new classes are needed?

Reply:
Extra `"` after `description_complete` class too

Reply:
Oh sorry... I missed it. So the reason for these new classes was -- `task`, `:` and `description` are now rendered as separate elements. So without them, sometimes the `:` is not staying inline, while other times, all three of them go to the next line. So I thought I would enclose them within tags to make them inline separately.